wikipendium/wiki/static/js/script.js

```diff
@@ -107,6 +107,15 @@ $(function(){
         $(tabId).addClass('active');
     });
 
+    $(window).on('hashchange', function(e) {
+      // Impressively, this ensures scrolling doesn't break (as much) in Safari
+      $('.toc').toggleClass('yolo');
```

Comment:
Consider giving the class a more understandable name, e.g. `safari-scroll-unbreak-hack` or similar instead of `yolo`.